lib/alephant/broker/response/base.rb

```diff
@@ -61,6 +63,10 @@ def add_etag_allow_header
           })
         end
 
+        def self.options?(request_env)
+          request_env && request_env.respond_to?(:options?) && request_env.options?
```

Comment:
Heh, would be nice to use that feature (http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/) but we're currently targeting older builds of ruby. :)